.npmignore

```diff
@@ -0,0 +1,6 @@
+# Ignore all
+*
+
+# But not these files
+!src/**/*
```

Comment:
do we really want to publish library sources?

It seems like we have to exclude `dist` folder

Since in package.json:
```
  "main": "dist/index.js",`
```